website/docs/language/resources/provisioners/connection.mdx

```diff
@@ -174,6 +174,21 @@ The `ssh` connection also supports the following fields to facilitate connnectio
   must be used in conjunction with a `bastion_private_key`. These can be loaded from
   a file on disk using the [the `file` function](/language/functions/file).
 
+## Connection through a HTTP Proxy with SSH
+
+The `ssh` connection also supports the following fields to facilitate connections by SSH over HTTP proxy.
+
+* `proxy_scheme` - http or https
+
+* `proxy_host` - Setting this enables the SSH over HTTP connection. This host
+  will be connected to first, and then the `host` or `bastion_host` connection will be made from there.
+
+* `proxy_port` - The port to use connect to the proxy host.
+
+* `proxy_user_name` - The username to use connect to the private proxy host.
+
+* `proxy_user_password` - The password to use connect to the private proxy host.
```

Comment:
We should clarify which of these fields are optional for the HTTP proxy to be used.

Reply:
Fixed